$OpenBSD: patch-mlib_include_mTextParam_h,v 1.1.1.1 2019/12/07 02:06:45 bcallah Exp $

Fix occurrences of code where char is considered as signed on all platforms.

Index: mlib/include/mTextParam.h
--- mlib/include/mTextParam.h.orig
+++ mlib/include/mTextParam.h
@@ -27,7 +27,7 @@ extern "C" {
 typedef struct _mTextParam mTextParam;
 
 void mTextParamFree(mTextParam *p);
-mTextParam *mTextParamCreate(const char *text,char split,char splitparam);
+mTextParam *mTextParamCreate(const char *text,char split,signed char splitparam);
 
 mBool mTextParamGetInt(mTextParam *p,const char *key,int *dst);
 mBool mTextParamGetInt_range(mTextParam *p,const char *key,int *dst,int min,int max);
